xdsl/dialects/func.py

```diff
@@ -11,9 +11,7 @@
     SymbolRefAttr,
 )
 from xdsl.dialects.utils import (
-    parse_call_op_like,
```

Comment:
Let's delete these helpers also?

Reply:
ehm, I have?

Reply:
I don't see a diff in `xdsl.dialects.utils`

Reply:
they are still used in `riscv_func`, so maybe after this :)